vars/buildPlugin.groovy

```diff
@@ -274,12 +284,14 @@ def call(Map addonParams = [:])
 						def packageversion
 						def changespattern = [:]
 						def dists = params.dists.tokenize(',')
-						def ppas = params.PPA == "auto" ? [PPA_VERSION_MAP[version].each{p -> PPAS_VALID[p]}].flatten() : []
+						def ppas = params.PPA == "auto" ? (PPA_VERSION_MAP.containsKey(version) ? [PPA_VERSION_MAP[version].each{p -> PPAS_VALID[p]}].flatten() : []) : []
```

Comment:
untestedm but I guess that should work while being more readable
```suggestion
						def ppas = (params.PPA == "auto"  && PPA_VERSION_MAP.containsKey(version)) ? [PPA_VERSION_MAP[version].each{p -> PPAS_VALID[p]}].flatten() : []
```

Reply:
yea it looks better, thanks